docs/v3/core/index.rst

```diff
@@ -1205,40 +1070,47 @@ the following procedure:
 
 4. The chunk array ``A`` is equal to ``EC[0]``.
 
-Specifying codecs
------------------
+.. _codec-specification:
+
+Core codecs
+-----------
+
+This specification defines a set of codecs ("core codecs") which all Zarr implementations SHOULD implement in
```

Comment:
Where is this list, actually?

Reply:
I interpret this as anything listed on https://zarr-specs--330.org.readthedocs.build/en/330/v3/codecs/index.html

Reply:
I think in a previous draft we had just `bytes` and `transpose` as core codecs.

Reply:
Opened an issue: https://github.com/zarr-developers/zarr-specs/issues/341